client/src/test/java/com/graphhopper/api/GraphHopperWebIT.java

```diff
@@ -55,6 +56,25 @@ public void testSimpleRoute() {
     }
 
     @Test
+    public void testTimeout() {
+        GHRequest req = new GHRequest().
+                addPoint(new GHPoint(49.6724, 11.3494)).
+                addPoint(new GHPoint(49.6550, 11.4180));
+        GHResponse res = gh.route(req);
+        assertFalse("errors:" + res.getErrors().toString(), res.hasErrors());
+
+        req.getHints().put(GraphHopperWeb.TIMEOUT, 1);
```

Comment:
Ah, no, good idea. Then probably put an assert in the exception block and the fail directly after the method call?